code/src/iris/src/lib/Analytics.svelte

```diff
@@ -194,13 +195,70 @@
       const startDate = new Date(now.getTime() - days * 24 * 60 * 60 * 1000);
       const dateFilter = startDate.toISOString().split('T')[0];
 
-      const response = await api.getInstallationDirspaces($selectedInstallation.id, {
-        q: `created_at:${dateFilter}..`,
-        tz: Intl.DateTimeFormat().resolvedOptions().timeZone,
-        limit: 200
-      });
+      // Load a reasonable sample for analytics (up to 10 pages / 500 runs)
+      const allDirspaces: Dirspace[] = [];
+      let hasMore = true;
+      let nextPageUrl: string | null = null;
+      let pagesLoaded = 0;
+      const maxPages = 10;  // Limit to 10 pages for performance
+      const timezone = Intl.DateTimeFormat().resolvedOptions().timeZone;
+      
+      while (hasMore && pagesLoaded < maxPages) {
+        let response;
+        
+        if (nextPageUrl) {
+          // Use the URL from Link header directly
+          const fetchResponse = await fetch(nextPageUrl, {
+            method: 'GET',
+            headers: { 'Content-Type': 'application/json' },
+            credentials: 'include',
+          });
+          
+          const rawResponse = await fetchResponse.json();
+          
+          // Parse Link headers from the response
+          const linkHeader = fetchResponse.headers.get('Link') || fetchResponse.headers.get('link');
```

Comment:
The `get` method is case insensitive.

https://developer.mozilla.org/en-US/docs/Web/API/Headers/get

> The name of the HTTP header whose values you want to retrieve from the Headers object. If the given name is not the name of an HTTP header, this method throws a [TypeError](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/TypeError). **The name is case-insensitive.**